pkg/shp/cmd/build/create.go

```diff
@@ -0,0 +1,100 @@
+package build
+
+import (
+	"errors"
+
+	buildv1alpha1 "github.com/shipwright-io/build/pkg/apis/build/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/shipwright-io/cli/pkg/shp/cmd/runner"
+	"github.com/shipwright-io/cli/pkg/shp/params"
+	"github.com/spf13/cobra"
+)
+
+// CreateSubCommand contains data input from user
+type CreateSubCommand struct {
+	url      string
+	strategy string
+	name     string
+
+	image string
+
+	build *buildv1alpha1.Build
+
+	cmd *cobra.Command
+}
+
+func createSubCmd() runner.SubCommand {
+	c := &cobra.Command{
+		Use:   "create [flags] name strategy url",
+		Short: "Create Build",
+	}
+
+	c.Flags().StringP("output-image", "i", "", "Output image created by build")
+
+	return &CreateSubCommand{
+		cmd: c,
+	}
+}
+
+// Cmd returns cobra Command object of the create subcommand
+func (sc *CreateSubCommand) Cmd() *cobra.Command {
+	return sc.cmd
+}
+
+// Complete fills internal subcommand structure for future work with user input
+func (sc *CreateSubCommand) Complete(params params.Params, args []string) error {
+
+	if len(args) < 3 {
+		return errors.New("Not enough arguments for Build create")
```

Comment:
If I recall correctly (but don't quote me), we usually start errors with a lower case letter?
@adambkaplan Correct me if I am wrong (maybe that was just an OpenShift thing ...)

Reply:
This makes sense, as error message starts with something that looks like `ERROR:`, so it's better to start it with lower case, thank you.